### PR TITLE
SMTP Client authentication error: Add 4th solution

### PR DESCRIPTION
**Pull request by gcaringal**
I came across following error when trying to send an email via SMTP: "5.7.57 Client not authenticated to send mail"
None of the 3 currently listed options worked. There is no mention in the article regarding a Conditional Access policy blocking legacy authentication.

I therefore propose to add a 4th option which is what helped me fix the error.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 1]
```diff
@@ -156,6 +156,16 @@ There are a few things you should check:
...
 
+4. Exclude the user from a [Conditional Access policy](active-directory/conditional-access/overview) that [blocks Legacy Authentication](azure/active-directory/conditional-access/block-legacy-authentication):
+
+      - Sign in to the Azure portal as a Security administrator, Conditional Access administrator, or Global administrator.
+
+      - Browse to **Azure Active Directory** > **Security** > **Conditional Access**
+
+      - In the policy that blocks Legacy Authentication, exclude the mailbox being used under **Users and Groups** > **Exclude**
+
+      - Select **Save**
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:167]:
```suggestion
      - Browse to **Azure Active Directory** > **Security** > **Conditional Access**.

      - In the policy that blocks Legacy Authentication, exclude the mailbox being used under **Users and Groups** > **Exclude**.

      - Select **Save**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gcaringal, kindly commit the minor suggested changes. Many thanks.

**Comment by gcaringal** [comment]
Thanks for the copy review @JohanFreelancer9.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**